src/slowmokit/methods/preprocessing/label_encoder.cpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file methods/preprocessing/label_encoder.hpp
+ *
+ * Implementation of label encoder
+ */
+
+#include "label_encoder.hpp"
+
+template<class T>
+std::vector<int> label_encoder(std::vector<T>& input ){
```

Comment:
Variables and function names should be in camel case

Reply:
oh , changing them

Reply:
changed sir